Delicate-Butterflies/Bangazon-Site
# Mb readme

Related Issue:
Description:
 More info for getting started

Chai/Mocha Tests File:

PR Testing Steps:
- view readme - specifically getting started section
- view overall  - see if any additional info is needed


Check list before you make a PR:
